### PR TITLE
Cl refactor4

### PR DESCRIPTION
**Pull request by cheeleertr**
## Type of Change

- [x] Debugging
- [x] New Feature
- [x] Refactor/File Clean-up

## Describe changes
Add Merchant Index page for easy navigating
Refactor navbar to have pertinent links only
Removed most unnecessary comments
Refactor invoice test
Debug and refactor Rake Task to work: add status to items, disable/enable foreign key restraint to destroy and load records"

## Checklist

- [x] I have made RSpec test  
- [x] My code is passing all tests 
- [x] I assigned a review for my PR

### CODE REVIEW

**Comment by cheeleertr** [comment]
I also added a Merchant Index to allow us to navigate the site without manually typing url.

File: app/views/shared/_header.html.erb [hunk 1]
```diff
@@ -4,8 +4,7 @@
 </div>
 <nav>
   <ul>
-    <li><%= link_to 'Dashboard', '#' %></li>
-    <li><%= link_to 'Merchants', '#' %></li>
-    <li><%= link_to 'Invoices', '#' %></li>
+    <li><%= link_to 'Admin Dashboard', admin_index_path %></li>
+    <li><%= link_to 'Merchants Index', merchants_path %></li>
   </ul>
```
Comment (jdmchugh111) [inline app/views/shared/_header.html.erb:9]: good change! makes site much more easy to navigate

File: config/routes.rb [hunk 2]
```diff
@@ -6,30 +6,23 @@
...
-  resources :merchants, only: [:index, :show, :edit, :new, :create]
-  resources :invoices, only: [:index, :show]
-  resources :invoice_items, only: [:update]
-end
-
-patch "/admin/merchants/:id", to: "admin/merchants#update", as: "admin_merchant_update"
+  namespace :admin do
+    resources :merchants, only: [:index, :show, :edit, :new, :create]
+    resources :invoices, only: [:index, :show]
+  end
```
Comment (jdmchugh111) [inline config/routes.rb:24]: resources page looks good!

File: lib/tasks/csv_load.rake [hunk 3]
```diff
@@ -2,59 +2,96 @@
...
+    disable_foreign_keys
+    Transaction.destroy_all
     transactions_path = "./db/data/transactions.csv"
     CSV.foreach(transactions_path, headers: true) do |row|
       Transaction.create!(row.to_hash)
     end
     ActiveRecord::Base.connection.reset_pk_sequence!('transactions')
+    enable_foreign_keys
+    puts "Successfully loaded transactions"
   end
```
Comment (jdmchugh111) [inline lib/tasks/csv_load.rake:95]: good work fixing the rake tasks

File: spec/features/admin_invoices/show_spec.rb [hunk 4]
```diff
@@ -102,34 +89,13 @@
...
-
-    # @invoice32 = FactoryBot.create(:invoice, customer: @customer10, status: 2)
-    # @invoice33 = FactoryBot.create(:invoice, customer: @customer10, status: 2)
-    # @invoice34 = FactoryBot.create(:invoice, customer: @customer10, status: 2)
     ]
 
-    # @transaction1 = FactoryBot.create(:transaction, invoice: @invoice1, result: "success")
-    # @transaction2 = FactoryBot.create(:transaction, invoice: @invoice1)
-    # @transaction3 = FactoryBot.create(:transaction, invoice: @invoice1)
-
```
Comment (jdmchugh111) [inline spec/features/admin_invoices/show_spec.rb:132]: awesome job cleaning all of this up

**Review by jdmchugh111 — APPROVED** [review]
great job! really went the extra mile on this

**Pull request merged**